azure-pipelines.yml

```diff
@@ -406,24 +406,7 @@ jobs:
         - sw_build
   - bash: |
       set -e
```

Comment:
More Nits, just things you should know they don't need fixing here:  I think a 1 line script doesn't need set -e to return the return value of the solitary line.